components/scream/data/scream_input.yaml

```diff
@@ -18,4 +18,118 @@ Atmosphere Driver:
     Reference Grid: Physics GLL
     Dynamics Driven:
       Dynamics Namelist File Name: ./data/theta.nl
+
+SCREAM:
+
+  #=====================
+  # I'm confused how we should set this. You shouldn't be able
+  # to change this once a run starts, but you should be able to choose it
+  # when you start a new run... What if your initial condition is for year 2000
+  # but you're doing a control run and want it to start at 01-01-0001?
+  Start Date: 01-01-2000
```

Comment:
Can this be gathered from `case_run.xml` and thus replaced by the `./create_case` call?

Reply:
@AaronDonahue , if start date comes from the CIME XML env, I'm wondering if we even need this in the YAML? I guess maybe for non-CIME (aka standalone) runs? Normal CIME cases use this setting:

```
<entry id="RUN_STARTDATE" value="0001-01-01">
```

Reply:
Ok, I think I know a good solution. Commit coming.

Reply:
Should we remove the comment, since you found a way to do it?